sdk/storage/azure-storage-blob-batch/src/samples/java/com/azure/storage/blob/batch/ReadmeSamples.java

```diff
@@ -23,12 +23,12 @@
  * Code samples for the README.md
  */
 public class ReadmeSamples {
-    private BlobServiceClient blobServiceClient = new BlobServiceClientBuilder().buildClient();
+    private BlobServiceClient blobServiceClient = new BlobServiceClientBuilder().endpoint("<YOUR_END_POINT>").sasToken("<YOUR_SAS_TOKEN>").buildClient();
```

Comment:
Need to regenerate the README codesnippets

Reply:
How do I do that?

Reply:
None of this should change the lines of code the following snippets are on so I think its ok right?